doc/reference/reference_sql/sql_user_guide.rst

```diff
@@ -935,7 +951,13 @@ From STRING to INTEGER or UNSIGNED is allowed only if the string has a represent
 and the result is not out of range,
 and the number has no post-decimal digits. |br|
 From STRING to DOUBLE or NUMBER is allowed only if the string has a representation of a number. |br|
-From BOOLEAN to number is allowed only if the number is not DOUBLE.
+From BOOLEAN to number is allowed only if the number is not DOUBLE. |br|
+From STRING to UUID is allowed only if the value is
+(8 hexadecimal digits) hyphen (4 hexadecimal digits) hyphen (4 hexadecimal digits) hyphen (4 hexadecimal digits) hyphen (12 hexadecimal digits),
+such as ``'8e3b281b-78ad-4410-bfe9-54806a586a90'``. |br|
+From VARBINARY to UUID is allowed only if the value is
+(32 hexadecimal digits),
+such as ``X'8e3b281b78ad4410bfe954806a586a90'``. |br|
```

Comment:
Sorry, I didn't think about this part last time. I believe that this part is not just about literals. If so, should we mention that a VARBINARY can only be cast to a UUID if its length is 16? Thus, the literal for such a value can be represented using 32 hexadecimal digits. There are a few more conditions (which I wrote about last time), but I'm not sure if we should mention them here.

Reply:
Would you prefer:
From VARBINARY to UUID is allowed only if the value is 16 bytes long, such as ``X'8e3b281b78ad4410bfe954806a586a90'``.
?

Reply:
Yes, it looks good.

Reply:
Okay, a change has been committed.